### PR TITLE
LW-10578 - Shared wallet creation

### PR DESCRIPTION
**Pull request by danielmain**
This creates a new shared wallet

### CODE REVIEW

File: apps/browser-extension-wallet/src/hooks/useWalletManager.ts [hunk 1]
```diff
@@ -450,6 +453,43 @@ export const useWalletManager = (): UseWalletManager => {
     [updateAppSettings, setCardanoWallet, setCurrentChain]
   );
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const generateSharedKeys = async ({
```
Comment (pczeglik-iohk) [inline apps/browser-extension-wallet/src/hooks/useWalletManager.ts:458]: It looks this function is not used. At least GitHub Actions says it 👇 

Should it be exported maybe?

Comment (pczeglik-iohk) [reply apps/browser-extension-wallet/src/hooks/useWalletManager.ts:458]: Why do you use `@ts-ignore` @danielmain

File: apps/browser-extension-wallet/src/hooks/useWalletManager.ts [hunk 2]
```diff
@@ -34,6 +34,8 @@ import { BackgroundService } from '@lib/scripts/types';
 import { getChainName } from '@src/utils/get-chain-name';
 import { useCustomSubmitApi } from '@hooks/useCustomSubmitApi';
 import { setBackgroundStorage } from '@lib/scripts/background/storage';
+import { KeyPurpose } from '@cardano-sdk/key-management/dist/cjs/types';
```
Comment (pczeglik-iohk) [inline apps/browser-extension-wallet/src/hooks/useWalletManager.ts:37]: Isn't this type available to be imported straight from `'@cardano-sdk/key-management`

File: apps/browser-extension-wallet/src/hooks/useWalletManager.ts (hunk 1 above)
Comment (danielmain) [reply apps/browser-extension-wallet/src/hooks/useWalletManager.ts:458]: Hi @pczeglik-iohk thank for reviewing it :) 

Just I forgot to add the WIP label. So this PR is far from being ready. That ts-ignore is only to pass the commit/push hook 

regards,
@danielmain

**Comment by greatertomi** [comment]
This is already done on #1268

**Pull request closed without merging**